app/models/archive.rb

```diff
@@ -10,12 +10,13 @@ def initialize(year:, month:, day: nil, page: 1)
     @year  = year
     @month = month
     @page  = page
+    @day   = day
   end
 
   def articles
     return @articles if defined?(@articles)
 
-    @articles = Article.published.live.root.all
+    @articles = Article.published.live.root.order(title: :desc)
```

Comment:
Will this order all Articles alphabetically by title even on collections that are more than just a day?